functions/v2/deploy-function/deploy_function.py

```diff
@@ -0,0 +1,68 @@
+# [START create_cloud_function_v2]
+from google.cloud.functions_v2 import FunctionServiceClient, CreateFunctionRequest
+from google.cloud.functions_v2.types import (
+    Function,
+    BuildConfig,
+    ServiceConfig,
+    StorageSource,
+    Source,
+)
+
+
+def create_cloud_function(
+    project_id,
+    bucket_name,
+    location_id,
+    function_id,
+    entry_point,
+    function_archive,
+    runtime="python311",
+):
+    client = FunctionServiceClient()
+    parent = f"projects/{project_id}/locations/{location_id}"
+    function_name = (
+        f"projects/{project_id}/locations/{location_id}/functions/{function_id}"
+    )
+    function = Function(
+        name=function_name,
+        build_config=BuildConfig(
+            source=Source(
+                storage_source=StorageSource(
+                    bucket=bucket_name,
+                    object_=function_archive,
+                    generation=2,
+                )
+            ),
+            runtime=runtime,
+            entry_point=entry_point,
+        ),
+        service_config=ServiceConfig(available_memory="256M"),
+    )
+
+    request = CreateFunctionRequest(
+        parent=parent, function=function, function_id=function_id
+    )
+
+    operation = client.create_function(request=request)
+    print("Waiting for operation to complete...")
+    response = operation.result()
+    print(response)
```

Comment:
Probably would be better to print certain attribute, which can indicate that job has successfully been created (only runtime, since you are checking it in tests)

Reply:
I'll change approach to testing. Lets keep it for sample